dali/dfu/dfurun.cpp

```diff
@@ -1418,8 +1418,7 @@ class CDFUengine: public CInterface, implements IDFUengine
                         Owned<IFileDescriptor> olddstf;
                         if (diffNameSrc.get()||diffNameDst.get())
                         {
-                            Owned<IFileDescriptor> newf;
-                            newf.setown(srcFile->getFileDescriptor());
+                            Owned<IFileDescriptor> newf = srcFdesc.get();
```

Comment:
this is -ve leak (i.e. this will incorrectly Release and ultimately could cause a crash as free too early).

Why have 'newf' at all now? just use srcFDesc.

Reply:
Changed